gallery.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0"?>
+
+<plugin category="generic" product="webOfScience">
+
+  <name locale="en_US">Web of Science Reviewer Recognition</name>
+
+  <homepage>https://github.com/clarivate/wos_reviewer_recognition_plugin_ojs_3/</homepage>
+
+  <summary locale="en_US">
+    This plugin enables integration with Web of Science Reviewer Recognition Service. To install and use this plugin you must first purchase the Web of Science Reviewer Recognition Service subscription.
+  </summary>
+
+  <description locale="en_US">
+    <![CDATA[<p>Web of Science Reviewer Recognition Service is a third-party workflow solution allowing you to provide your reviewers with the opportunity to gain recognition for their reviews on Web of Science.</p><p>To install and use this plugin you must first purchase the Web of Science Reviewer Recognition Service. To do this please register your interest by <a href=\"mailto:reviewservices@clarivate.com\">emailing Clarivate</a>. The Clarivate team will then be in touch to discuss your options and provide further instructions on how to integrate your journal with Web of Science Reviewer Recognition.</p>]]>
+  </description>
+
+  <maintainer>
+    <name>Clarivate</name>
+    <institution>Clarivate</institution>
+    <email>reviewservices@clarivate.com</email>
+  </maintainer>
+
+  <release date="2024-08-05" version="1.3.4.0" md5="71d1dbed08050f4eb589927f02d8e4ab">
+
+    <package>
+      https://github.com/publons/ojs_3_plugin/releases/download/2021.5.25.0/publons.tar.gz
```

Comment:
I assume this will be updated once we do the release?

Reply:
Yep, date and md5 will need to be updated.